### PR TITLE
About Page Created

### PR DESCRIPTION
**Pull request by Chandan-SM**
- About Page created.

- Changes made as suggested as asked in the previous pr.

- Slight changes in NavBar Css.

- Backend Fetch Implemented

### CODE REVIEW

File: score_keeper_client/src/components/OldGame.jsx [hunk 1]
```diff
@@ -1,37 +1,28 @@
...
-
-
+  const [player1, setPlayer1] = useState('');
+  const [player2, setPlayer2] = useState('');
+  const [score1, setScore1] = useState(0);
+  const [score2, setScore2] = useState(0);
+  const [winText, setWinText] = useState('');
+
+  useEffect(() => {
+    fetch('http://localhost:4000/singleGameData/64145d0bb0a5c251d35886e5')
```
Comment (PatelNeelMahesh) [inline score_keeper_client/src/components/OldGame.jsx:15]: Should not keep `http://localhost:4000` here, should use environment variable to store it. Later when you deploy your webapp, it becomes easier to replace the url and update it with domain name.

**Review by PatelNeelMahesh — COMMENTED** [review]
After making the changes you can merge pull request.

File: score_keeper_client/src/components/OldGame.jsx (hunk 1 above)
Comment (Chandan-SM) [reply score_keeper_client/src/components/OldGame.jsx:15]: Resolved.

**Comment by Chandan-SM** [comment]
About Page created.

Changes made as suggested as asked in the previous pr.

Slight changes in NavBar Css.

Backend Fetch Implemented

**Pull request merged**